### PR TITLE
xz 5.4.5

### PR DESCRIPTION
**Pull request by ik11235**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  +++ actual
  @@ -1 +1,3 @@
  -"135c90b934aee8fbc0d467de87a05cb70d627da36abe518c357a873709e5b7d6"
  +# encoding: US-ASCII
  +#    valid: true
  +"544d29069b006f167cc2f8e0339079abf8b9fcc775056fbf44015c[61](https://github.com/Homebrew/homebrew-core/actions/runs/6720576125/job/18265022159?pr=153048#step:3:62)2e8bae84"
```

**Comment by ik11235** [comment]
The URL in `archive.org` was removed because it does not function as a mirror.

A 404 is returned when opened in a browser.

File: Formula/x/xz.rb [hunk 1]
```diff
@@ -4,11 +4,11 @@ class Xz < Formula
   desc "General-purpose data compression with high compression ratio"
   homepage "https://tukaani.org/xz/"
   # The archive.org mirror below needs to be manually created at `archive.org`.
```
Comment (p-linnane) [inline Formula/x/xz.rb:6]: @ik11235 We need to manually upload the tarball to archive.org. I've done that and pushed some changes here.

**Comment by p-linnane** [comment]
Why did you change the build args back? I changed them intentionally.

File: Formula/x/xz.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/x/xz.rb:6]: 👍

Comment (ik11235) [reply Formula/x/xz.rb:6]: @p-linnane 

Thanks for the advice!

I have confirmed the upload to archive.org and have reinstated the mirror.
I also confirmed that `brew test xz` succeeds in the local environment.

**Comment by ik11235** [comment]
> Why did you change the build args back? I changed them intentionally.

Sorry, I did not mean to put it back in.
It seems I accidentally put it back in during the rebase.
I have reincorporated your changes.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@ik11235 no need to do forced update.

**Comment by chenrui333** [comment]
another couple of days run for linux build.

**Comment by chenrui333** [comment]
```
brew test --retry --verbose ammonite-repl
brew install --only-dependencies build2
brew link pkgconf
brew test --retry --verbose cucumber-cpp
brew test --retry --verbose git-svn
brew test --retry --verbose licensed
brew test --retry --verbose rpm
brew test --retry --verbose seal
brew test --retry --verbose sslmate
brew test --retry --verbose trafilatura
brew test --retry --verbose youtube-dl
```

**Pull request merged**